share/views/layouts/main.tt

```diff
@@ -168,7 +168,7 @@
           [% ELSE %]
             <i class="icon-user"></i>
           [% END %]&nbsp;
-            [% session.logged_in_user | html_entity %] <b class="caret"></b>
+            [% session.logged_in_fullname | html_entity %] <b class="caret"></b>
```

Comment:
I think we also want to fall back to username if the full name is not available (I believe full name is optional?).

Reply:
I will check if I can have a solution for this point :)